WordPress/Classes/ViewRelated/Jetpack/Branding/Overlay/JetpackOverlayViewController.swift

```diff
@@ -0,0 +1,50 @@
+import UIKit
+
+class JetpackOverlayViewController: UIViewController {
+
+    private var redirectAction: (() -> Void)?
+
+    private var viewFactory: ((() -> Void)?) -> UIView
+
+    init(viewFactory: @escaping ((() -> Void)?) -> UIView, redirectAction: (() -> Void)? = nil) {
```

Comment:
And here we can just pass the UIView if I am not missing something.

Reply:
We could, but the intent is to make the viewController agnostic of the specific type. It's not necessary, but if we decide to change the view, this would work without changes.